PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -38,7 +38,7 @@ class Generator(HasTraits):
     meanEventNumber = Float(2)
     scaleFactor = Float(2) # Note: we don't expose the scale factor in the view
     meanTime= Float(2000)
-    mode = Enum(['STORM','PAINT'])
+    mode = Enum(['STORM','DNA PAINT'])
```

Comment:
```suggestion
    mode = Enum(['STORM','PAINT'])
```

The DNA is redundant and  overly specific as there are multiple forms of non-DNA PAINT (e.g. with WGA)

Reply:
I was under the impression that not all PAINT forms are described by reversible binding/unbinding acording to first order kinetics. Do they all follow the same statistics?

Reply:
In general I think they are all *supposed* to be reversible binding/unbinding with 1st order kinetics (or a reasonable approximation thereof). I suspect some have quite a slow off-rate so in practice you bleach before the label comes off, but this would still look like 1st order kinetics from a simulation perspective.